src/test/java/edu/cornell/kfs/sys/xmladapters/TrimmedStringXmlAdapterTest.java

```diff
@@ -0,0 +1,62 @@
+package edu.cornell.kfs.sys.xmladapters;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import edu.cornell.kfs.concur.ConcurConstants.ConcurOAuth2.WebServiceCredentialKeys;
+import liquibase.repackaged.org.apache.commons.lang3.StringUtils;
```

Comment:
Please use the regular lang3 StringUtils import instead.

Reply:
uggg, thanks